modules/3d/test/test_utils.cpp

```diff
@@ -0,0 +1,71 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html
+
+// This code is also subject to the license terms in the LICENSE_WillowGarage.md file found in this module's directory
+
+#include "test_precomp.hpp"
+
+namespace opencv_test { namespace {
+
+class CV_RgbdDepthTo3dTest: public cvtest::BaseTest
+{
```

Comment:
> cvtest::BaseTest

We really don't need that here.

Reply:
Fixed, got rid of all `cvtest` stuff in tests